packages/expo-cli/src/easJson.ts

```diff
@@ -88,6 +92,7 @@ const AndroidGenericSchema = Joi.object({
   credentialsSource: Joi.string().valid('local', 'remote', 'auto').default('auto'),
   gradleCommand: Joi.string(),
   artifactPath: Joi.string(),
+  releaseChannel: Joi.string(),
```

Comment:
so shouldn't we set the default?
```suggestion
  releaseChannel: Joi.string().default('default'),
```